src/fontrender.c

```diff
@@ -15,8 +15,21 @@
 
 const struct glyph *font_get_glyph(const struct font *font, glyph_t glyph)
 {
-	if (glyph > font->max) return NULL;
-	return font->glyphs[glyph];
+	if (glyph > font->max)
+		return NULL;
+	
+	size_t first = 0, last = font->count;
```

Comment:
`font->count` and `font->max` should be the same value now (perhaps offset by 1), yes? Is it worth consolidating?

Reply:
No. For example, if font contains two characters, 'a' and 'z', `font->count` will be 2, but `font->max` will be 0x7a (code point of 'z').

Reply:
Ah, so it still tracks the code point, not its index. Got it.